### DISCUSSION TITLE
How to  add topics to org repos

### DISCUSSION THREAD

**Discussion by satishbkodali** [octokit/octokit.js#1765] [Q&A] [answered]
Currently we have this octokit.repos.replaceAllTopics which updates topics on the owner repos.
Is it possible to update topics for org repos as well?

**Comment by gr2m** [comment, marked as answer]
You can use `octokit.repos.replaceAllTopics` for all repositories that you have write permission to:

```js
octokit.repos.replaceAllTopics({
  owner: 'org-login',
  repo: 'repo-name',
  names: ['topic1', 'topic2', 'topic3']
})
```

Does that work for you?

**Reply by satishbkodali** [reply]
Thanks 
giving orgName in the owner worked .
